docs/uml-class-diagrams/database/production/DatabaseGisManagerImplementation/DatabaseGisManagerImplementation.puml

```diff
@@ -2,7 +2,14 @@
 
 package edu.oakland.production.database {
     class DatabaseGisManagerImplementation {
-        __
+    	- LocationDataPoint locDataPoint
+    	- String mode
+    	- String currentSat
+    	        __
+        + DatabaseGisManagerImplementation (LocationDataPoint locationDataPoint, 
+        \t String mode, 
+        \t String currentSat)
+        
```

Comment:
Does this class need access to private attributes? If not, they should be deleted.
```suggestion
        + DatabasePersistentStorage storage
        __
        + DatabaseGisManagerImplementation(DatabasePersistentStorage storage)
```